src/helper/enums.ts

```diff
@@ -0,0 +1,11 @@
+export var AppointmentTypes = {
```

Comment:
If the data will not change in code use const.
Else use let.

var is not so used anymore because of its way of refering variables